lib/BlockedBy.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+const NodeResque = require('node-resque');
+const { queuePrefix } = require('../config/redis');
+
+class BlockedBy extends NodeResque.Plugin {
+    async beforePerform() {
+        const { jobId, blockedBy } = await this.jobInfo();
```

Comment:
`NodeResque.plugin` seems to pass job information into the plugin (https://github.com/taskrabbit/node-resque/blob/master/lib/plugin.js#L4-L11). Why not put blockedBy information in the queue and just use `this.job.blockedBy` here to save redis calls?

Reply:
@pranavrc Then we would need to modify the `args`. Currently, the `args` is only `buildId`, then we look up based on that inside `start`. Do you mean to modify this to change the `args`: https://github.com/screwdriver-cd/executor-queue/blob/master/index.js#L284-L286

Reply:
Yeah, just wondering if it would get real expensive to do a hget on every single item in the queue in a recurring fashion.

Reply:
Makes sense. Yeah I can also change the other side to pass in the information directly 👍